### PR TITLE
InstData Generate Cmds: check if file is empty

### PR DESCRIPTION
**Pull request by ebelter**

### CODE REVIEW

File: lib/perl/Genome/InstrumentData/Command/Import/GenerateBase.pm [hunk 1]
```diff
@@ -81,9 +81,10 @@ sub _open_file_parser {
         });
     die $self->error_message('Failed to create Text::CSV parser!') if not $parser;
 
+    die $self->error_message('File (%s) is empty!', $file) if not -s $file;
     my $fh = Genome::Sys->open_file_for_reading($file);
     my $headers = $parser->getline($fh);
-    die $self->error_message('File (%s) does not have any headers! Is it empty?', $file) if not $headers;
+    die $self->error_message('File (%s) does not have any headers! Is is empty?', $file) if not $headers;
```
Comment (tmooney) [inline lib/perl/Genome/InstrumentData/Command/Import/GenerateBase.pm]: The `Is it empty?` sentence could be removed completely--we now know it's not.

Comment (acoffman) [inline lib/perl/Genome/InstrumentData/Command/Import/GenerateBase.pm]: Either way, it currently says `is is empty` so that's probably not correct!

**Comment by acoffman** [comment]
+1 with typo fix!

**Comment by ebelter** [comment]
I removed the check all together. The headers are checked later on anyway.

**Comment by tmooney** [comment]
+1 :monkey:

**Comment by iferguson90** [comment]
+1 :boar:

**Pull request merged**